SharpBucket/V1/SharpBucketV1.cs

```diff
@@ -8,7 +8,7 @@ public sealed class SharpBucketV1 : SharpBucket{
       /// https://confluence.atlassian.com/display/BITBUCKET/Version+1
       /// </summary>
       public SharpBucketV1(){
-         _baseUrl = "https://bitbucket.org/api/1.0/";
+         BaseUrl = "https://bitbucket.org/api/1.0/";
```

Comment:
The same as before, let's leave it for now and maybe change it in another PR in all files.